### PR TITLE
Confluent Components 4.0 Release With Security

### PR DESCRIPTION
**Pull request by cyan-confluent**

### CODE REVIEW

File: repo/packages/C/confluent-connect/10/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
```
Comment (ryadav88) [inline repo/packages/C/confluent-connect/10/package.json]: change `packagingVersion` to 4.0

File: repo/packages/C/confluent-connect/10/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "confluent-connect",
+  "version": "1.1.0-4.0.0",
+  "scm": "https://github.com/confluentinc/kafka",
+  "description": "Confluent Connect worker. Includes Replicator Connector.\n\n\tDocumentation: http://docs.confluent.io/4.0.0/connect/managing.html",
+  "maintainer": "partner-support@confluent.io",
+  "tags": ["kafka", "confluent", "connect"],
+  "preInstallNotes": "This DC/OS Service is currently in preview. Preparing to install confluent-connect",
```
Comment (ryadav88) [inline repo/packages/C/confluent-connect/10/package.json]: you can remove this line: `This DC/OS Service is currently in preview.`

File: repo/packages/C/confluent-connect/10/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "postInstallNotes": "confluent-connect has been installed.",
+  "postUninstallNotes": "confluent-connect was uninstalled successfully.",
+  "minDcosReleaseVersion" : "1.10",
```
Comment (ryadav88) [inline repo/packages/C/confluent-connect/10/package.json]: `minDcosReleaseVersion` should be 1.9

**Review by ryadav88 — CHANGES_REQUESTED** [review]
The review comments apply to all the packages here.

**Comment by cyan-confluent** [comment]
@ryadav88 Your requested changes have been submitted.

**Review by ryadav88 — APPROVED** [review]

**Pull request merged**